crates/doc/src/lib.rs

```diff
@@ -114,6 +114,7 @@ pub mod tuple_pack;
 // It's similar to (and built from) a JSON Schema, but includes only
 // those inferences which can be statically proven for all documents.
 pub mod shape;
+pub use shape::schema::to_schema;
```

Comment:
blah sorry. My `rust-analyzer` bugged out because of the upgrade in Rust versions (apparently, a `rustup upgrade` fixed it..) and it kept on trying to make me do this refactor. Removed 👍